var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -60,6 +60,12 @@ class Hypre(Package):
     variant('openmp', default=False, description='Enable OpenMP support')
     variant('debug', default=False,
             description='Build debug instead of optimized version')
+    variant('cuda-uvm', default=False,
```

Comment:
Are there configure args tied specifically associated with this variant?

Reply:
Same question for `curand` and `cub`.

Reply:
I see. Yes, so at this point there *really* is only the `cuda` variant itself. If `+cuda`, then we get that set of configure args turned on, otherwise we don't. I've pushed an additional commit to resolve this. Thank you.